src/components/Image/index.tsx

```diff
@@ -56,21 +52,19 @@ function Image(props) {
     );
 }
 
-function imagePropsAreEqual(prevProps, nextProps) {
+function imagePropsAreEqual(prevProps: ImageOwnProps, nextProps: ImageOwnProps) {
```

Comment:
NAB but idk why this function isn't just inlined as an arrow function since it's so simple